drivers/gles3/shaders/stdlib_inc.glsl

```diff
@@ -1,5 +1,9 @@
 
-#ifdef USE_GLES_OVER_GL
+// Compatibility renames. These are exposed with the "godot_" prefix
```

Comment:
I think we can do this a little cleaner. I don't like making users write "godot_" before these function calls. Its a recipe to introduce regressions. 

Let's look at the reasons why functions in this file exist:
1. lines 4-47 are introduced on desktop only because these functions are part of the GLSL es 300 spec but not the GLSL 330 spec. So they keep their original names, but are only added for destop devices
2. lines 54-76 are introduced because on newer Adreno devices the `pack_***_4x8` functions are wrongfully exposed resulting in a redefinition error. Since we shouldn't have access to those functions anyway, we force the compiler to use the ones defined by us.
3. With this PR, it seems that older Adreno devices have bugs with the packing functions, so we want to use our manually implemented ones instead. In order to use our manually implemented ones, we need t change the function names to avoid function name conflicts. 

I am okay with always falling back to the functions we defined (they shouldn't be any slower than the driver implemented versions and your testing confirmed that). However, since you have used #define to replace the functions with our internal functions, we should not be using the "godot_" prefix in shaders. Shaders should continue to use the normal name (i.e. `packHalf2x16` instead of `godot_packHalf2x16`)

Reply:
Yes I understand. Changes just committed! ;)